lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -137,12 +137,14 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
             system_identifier = INTERNAL_SYSTEM)
 
     if not patient_entry:
-        LOG.warning("Skipping record with insufficient information to construct patient")
+        LOG.warning("Skipping record with insufficient information to construct patient for record: "
+            f"{enrollment.get('record_id')}")
         return None
 
     birthdate = parse_date_from_string(enrollment.get('core_birthdate'))
     if not birthdate:
-        LOG.warning("Record has an invalid or missing `core_birthdate` value")
+        LOG.warning("Record has an invalid or missing `core_birthdate` value for record: "
+            f"{enrollment.get('record_id')}")
```

Comment:
Very minor, but the convention in other log messages in our codebase puts the id immediately adjacent to the noun, e.g.:

```python
LOG.warning(f"Record {enrollment.get('record_id')} has an invalid or missing `core_birthdate` value")
```

so that the message is more concise and reads as a natural sentence.

Reply:
That was a bit wordy! Fixed.